features/step_definitions/usuario_steps.rb

```diff
@@ -0,0 +1,71 @@
+Given('Um usuario administrador existe') do
+  Usuario.create(nome: 'admin', email: 'admin@example.com', telefone: '8799999999', data_nascimento: Date.new(1999, 9, 9), password: 'password', password_confirmation: 'password', tipo: 2)
+end
+
+And('Eu estou logado como administrador com email {string} e senha {string}') do |email, senha|
+  visit "/sign_in"
+  fill_in 'session[email]', :with => email
+  fill_in 'session[password]', :with => senha
+  click_button 'Entrar'
+  expect(page).to have_content('Admin')
+end
+
+And('Eu estou na pagina administrar usuarios') do
+  click_link 'Administrar outros usuarios'
```

Comment:
É sempre bom fazer a verificacao de que a pagina realmente é a esperada. Usando expect(page).to have_content()